tools/nuget/generate_nuspec_for_native_nuget.py

```diff
@@ -181,6 +181,7 @@ def generate_files(list, args):
     is_cpu_package = args.package_name in ['Microsoft.ML.OnnxRuntime', 'Microsoft.ML.OnnxRuntime.OpenMP']
     is_mklml_package = args.package_name == 'Microsoft.ML.OnnxRuntime.MKLML'
     is_cuda_gpu_package = args.package_name == 'Microsoft.ML.OnnxRuntime.Gpu'
+    is_tensorrt_package = args.package_name == 'Microsoft.ML.OnnxRuntime.TensorRT'
```

Comment:
I was thinking there won't be such a thing as a TensorRT package.
i.e. The ORT Gpu package will now also include TensorRT EP shared lib.

Reply:
It's just a intermediate package name not the final package name.

'Microsoft.ML.OnnxRuntime.TensorRT' is being used as intermediate name and this name is also the name of intermediate package:
https://github.com/microsoft/onnxruntime/blob/3303a832063ecbe8febfce366ffd105374879a16/tools/ci_build/github/azure-pipelines/nuget/templates/gpu.yml#L21
https://github.com/microsoft/onnxruntime/blob/tensorrt_ep_nuget_package_2/csharp/OnnxRuntime.CSharp.proj#L129

Reply:
ok thanks